### ISSUE TITLE
suggestion: aggressiveMerging with options

### ISSUE THREAD

**Issue by sandrina-p** [jakubpawlowicz/clean-css#811]
foo.scss

```
%one {
    font-size: 16px;
    width: 95%;
    margin: 10px;
}

%two {
    font-size: 1rem;
    width: calc(100% - 10px);
    margin: 15px;
}


.foo {
    @extend %one;
    @extend %two;
}
```

The output will be:
`.foo{font-size:1rem;width:calc(100% - 10px);margin:15px;}`

But i don't wanna to merge width: `calc()` and `rem` properties.
So my solution right now is the option `advanced: false;`

I would like to know if it's possible to say not merge specifics properties, ex: `calc()`and `rem`

If you wondering why, that's because I have these properties duplicated thinking about old browsers support.

Thanks

**Comment by jakubpawlowicz** [comment]
See #810 - we'll turn it off by default but I believe it's too granular to have a per-property control.

**Comment by jakubpawlowicz** [comment]
This is fixed per b809a36e7b56efc0ae6bac80135ea6d67853433b while whole aggressive merging will be gone in 4.0.
